source/moleculardefinition/moleculardefinition-examples-header.xml

```diff
@@ -13,7 +13,7 @@
 <h4>Code (Accession Number)</h4>
 <p>The following MolecularDefinitions show examples of a sequence represented as a code using an accession number.</p>
 <ul>
-    <li>(example file TBD)</li>
+    <li><a href="moleculardefinition-example-genomic-cyp2c19-accession.html">Genomic Sequence example of CYP2C19 By Accession Number</a></li>
```

Comment:
Please use "Example of Sequence profile of CYP2C19 as Accession Number" to improve consistency with the other examples

Reply:
Thanks, Done.